hedera-node/hedera-app/src/main/java/com/hedera/node/app/store/ReadableStoreFactory.java

```diff
@@ -153,13 +151,9 @@ private static Map<Class<?>, StoreEntry> createFactoryMap() {
      *
      * @param state the {@link State} to use
      */
-    public ReadableStoreFactory(@NonNull final State state) {
+    public ReadableStoreFactory(@NonNull final State state, Function<SemanticVersion, SoftwareVersion> versionFactory) {
```

Comment:
```suggestion
    public ReadableStoreFactory(@NonNull final State state, @NonNull final Function<SemanticVersion, SoftwareVersion> versionFactory) {
```
Here and in other places, just to match convention

Reply:
Fixed. And all other places as well.